### PR TITLE
Calendar page

### PR DESCRIPTION
**Pull request by ianlovell**

### CODE REVIEW

File: app/containers/Calendar.jsx [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+import { connect } from 'react-redux';
+import { getAllEvents } from '../actions';
 
 class Calendar extends Component {
 
+    componentDidMount() {
+        this.props.dispatch(getAllEvents());
+    }
+
+    handleEventSelected() {
```
Comment (jebbinSCL) [inline app/containers/Calendar.jsx]: Is this function meant to be empty?

File: app/reducers/events.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import {
```
Comment (jebbinSCL) [inline app/reducers/events.js]: Should the formatting and bracketing of this import be different from the rest of the imports?

File: app/reducers/events.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import immutable from 'immutable';
+import * as actionTypes from '../constants/actionTypes';
+
+const initialState = immutable.Map({
+    isFetching: false,
+    events: immutable.List(),
+});
+
+const events = handleActions({
+    [actionTypes.GET_EVENTS_START]: (state) => state.set('isFetching', true),
```
Comment (jebbinSCL) [inline app/reducers/events.js]: What is the effect of having the square brackets here?

File: app/services/EventsService.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import api from './ApiService.js';
+import immutable from 'immutable';
+
+const sessionsUri = 'http://api.bris.tech';
+
+class EventsService {
+    getAllEvents() {
+        const events = [{
+            id: 'df191161-7cf3-4230-9b19-d92bcb0dfe0c',
```
Comment (jebbinSCL) [inline app/services/EventsService.js]: Are the events meant to be duplicated between here and events-list.js?

File: app/containers/Calendar.jsx (hunk 1 above)
Comment (ianlovell) [inline app/containers/Calendar.jsx]: Yes, that will load the sessions of the selected event in a future pull request

File: app/services/EventsService.js (hunk 4 above)
Comment (ianlovell) [inline app/services/EventsService.js]: Yes, just some stub data until the gateway is in place

File: app/reducers/events.js (hunk 3 above)
Comment (ianlovell) [inline app/reducers/events.js]: Using square brackets to define a property name allows the use of variables - in this case the consts on the import object (actionTypes)

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**